sktime/forecasting/model_selection/_split.py

```diff
@@ -1051,12 +1051,33 @@ def get_n_splits(self, y: Optional[ACCEPTED_Y_TYPES] = None) -> int:
         n_splits : int
             The number of splits.
         """
+        from sktime.datatypes import check_is_scitype, convert_to
+
         if y is None:
             raise ValueError(
                 f"{self.__class__.__name__} requires `y` to compute the "
                 f"number of splits."
             )
-        return len(self.get_cutoffs(y))
+
+        scitype = ["Hierarchical", "Panel"]
+        is_non_single, _, metadata = check_is_scitype(y, scitype, True)
```

Comment:
to esure we compute only the metadata we need, the `return_metadata` arg should e the empty list

Reply:
empty list? but the return_metadata arg accepts a bool?

Reply:
you can also pass a list of str, which is as in the docstring. If you pass the empty list, only the types are returned.

Reply:
resolved [3623c7f](https://github.com/sktime/sktime/pull/5012/commits/3623c7ffb8ab791aace27ce32655c9c9375853a8)